tools/k8s-test/replicaset.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	v1beta1 "k8s.io/api/extensions/v1beta1"
+)
+
+func GetReplicaSet(ctx context.Context, name string) (*v1beta1.ReplicaSet, error) {
+	obj := new(v1beta1.ReplicaSet)
+	ok, err := getObject(ctx, "rs", name, obj, nil)
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, NotFound.New("no such replica set %q", name)
+	}
+	return obj, nil
+}
+
+func GetReplicaSetByOwner(ctx context.Context, name string, owner Object) (*v1beta1.ReplicaSet, error) {
+	obj := new(v1beta1.ReplicaSet)
+	ok, err := getObject(ctx, "rs", name, obj, &owner)
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, NotFound.New("no replica set %q owned by %s", owner, owner)
+	}
+	return obj, nil
+}
+
+func GetReplicaSets(ctx context.Context) ([]v1beta1.ReplicaSet, error) {
```

Comment:
it is never used, was it created for future uses?

Reply:
yep. will remove for now.